src/api/URLs.ts

```diff
@@ -39,13 +39,25 @@ class URLs {
     return url;
   }
 
-  public static cases(snoozeState: SnoozeState, receiptNumber?: string): URL {
+  public static cases(
+    filter: CaseSnoozeFilter,
+    receiptNumber?: string,
+    from?: Date,
+    to?: Date
+  ): URL {
     const url = URLs.casesBase();
-    url.pathname += "/" + snoozeState;
+    url.searchParams.append("mainFilter", filter);
```

Comment:
Still not a huge fan of that parameter name, but apparently I couldn't up up with anything else beforeI merged it.